scripts/templates/soccer.wbt.template

```diff
@@ -122,17 +122,17 @@ DEF SOCCER_FIELD Solid {
             -0.76 0 0.21
             -0.76 0 0.65
             -0.76 0 0.66
-            0.76 0.05 0.66
-            0.76 0.05 0.21
-            0.75 0.05 0.21
-            0.75 0.05 0.59
-            0.69 0.05 0.65
-            -0.69 0.05 0.65
-            -0.75 0.05 0.59
-            -0.75 0.05 0.21
-            -0.76 0.05 0.21
-            -0.76 0.05 0.65
-            -0.76 0.05 0.66
+            0.76 0.09 0.66
```

Comment:
@Adman these kinds of changes freak me out as I am not totally sure what we are changing but I trust you have it figured out :)